### PR TITLE
Reverse order for deposit txs

### PR DESCRIPTION
**Pull request by veado**
Part of #1287

### CODE REVIEW

File: src/renderer/services/chain/transaction/deposit.ts [hunk 1]
```diff
@@ -180,52 +180,52 @@ export const symDeposit$ = ({
         node: validateNode$()
       })
     ),
-    // 2. send RUNE deposit txs
+    // 2. send asset deposit txs
     liveData.chain<ApiError, SymDepositValidationResult, TxHash>((_) => {
-      setState({ ...getState(), step: 2, deposit: RD.progress({ loaded: 40, total }) })
+      setState({ ...getState(), step: 3, deposit: RD.progress({ loaded: 60, total }) })
```
Comment (veado) [inline src/renderer/services/chain/transaction/deposit.ts:185]: `step: 2`  ^

File: src/renderer/services/chain/transaction/deposit.ts [hunk 2]
```diff
@@ -180,52 +180,52 @@ export const symDeposit$ = ({
...
       const current = getState()
-      setState({ ...current, depositTxs: { ...current.depositTxs, rune: RD.success(txHash) } })
+      setState({ ...current, depositTxs: { ...current.depositTxs, asset: RD.success(txHash) } })
       return txHash
     }),
+    // 3. send RUNE deposit txs
     liveData.chain<ApiError, TxHash, TxHash>((_) => {
-      setState({ ...getState(), step: 3, deposit: RD.progress({ loaded: 60, total }) })
-      // 3. send asset deposit txs
+      setState({ ...getState(), step: 2, deposit: RD.progress({ loaded: 40, total }) })
```
Comment (veado) [inline src/renderer/services/chain/transaction/deposit.ts:209]: `step 3`

File: src/renderer/services/chain/transaction/deposit.ts [hunk 3]
```diff
@@ -180,52 +180,52 @@ export const symDeposit$ = ({
         node: validateNode$()
       })
     ),
-    // 2. send RUNE deposit txs
+    // 2. send asset deposit txs
     liveData.chain<ApiError, SymDepositValidationResult, TxHash>((_) => {
-      setState({ ...getState(), step: 2, deposit: RD.progress({ loaded: 40, total }) })
+      setState({ ...getState(), step: 2, deposit: RD.progress({ loaded: 60, total }) })
```
Comment (veado) [inline src/renderer/services/chain/transaction/deposit.ts:185]: `loaded: 40`

File: src/renderer/services/chain/transaction/deposit.ts [hunk 4]
```diff
@@ -180,52 +180,52 @@ export const symDeposit$ = ({
...
       const current = getState()
-      setState({ ...current, depositTxs: { ...current.depositTxs, rune: RD.success(txHash) } })
+      setState({ ...current, depositTxs: { ...current.depositTxs, asset: RD.success(txHash) } })
       return txHash
     }),
+    // 3. send RUNE deposit txs
     liveData.chain<ApiError, TxHash, TxHash>((_) => {
-      setState({ ...getState(), step: 3, deposit: RD.progress({ loaded: 60, total }) })
-      // 3. send asset deposit txs
+      setState({ ...getState(), step: 3, deposit: RD.progress({ loaded: 40, total }) })
```
Comment (veado) [inline src/renderer/services/chain/transaction/deposit.ts:209]: `loaded: 60`

**Review by thatStrangeGuyThorchain — APPROVED** [review]

**Pull request merged**